RefactorHomePage

ag3722 opened the pull request:
- Refactored Home Page UI

ag3722 committed:
RefactorHomePage

- Refactored Home Page UI

SaifA21 committed:
Add missing packages

ag3722 committed:
Update Home Page

- Integrated displaying merchant_name into UI
- Integrated Tab Bar filter functionality
- Integrated Issued & Redeemed Points into UI
- Integrated transactions data into chart UI
- Integrated Average Revenue, Net Input, and Net# Customer into UI

SaifA21 reviewed (changes requested):
Looks great! 

Just some padding issues:
- Need additional padding at the bottom of the date range selection
- Need to add padding at the very bottom such that the scroll view allows the user to scroll more. Currently the key metrics are sitting behind the nav bar.  

![Screenshot 2025-02-26 233924](https://github.com/user-attachments/assets/079042c7-1649-458a-b7e1-19c8721607d9)

ag3722 committed:
Fix Home Page Styling

- Added padding to bottom of tab bar
- Added padding to bottom of screen
- Added padding to right of key metrics

SaifA21 committed:
format + disable swipe on tabview

SaifA21 reviewed (approved):
LGTM
